mmdet/models/backbones/vit.py

```diff
@@ -0,0 +1,458 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+from mmcls.models import VisionTransformer as _VisionTransformer
+from mmcls.models.backbones.vision_transformer import \
+    TransformerEncoderLayer as _TransformerEncoderLayer
+from mmcls.models.utils import MultiheadAttention as _MultiheadAttention
+from mmcls.models.utils import resize_pos_embed
```

Comment:
Users may not install the mmcls codebase.

Reply:
try